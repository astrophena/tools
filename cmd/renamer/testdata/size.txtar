-- a.txt --
Hello, world!
-- b.txt --
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
-- c.txt --
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
