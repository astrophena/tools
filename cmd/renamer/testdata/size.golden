-- 1.txt --
Hello, world!
-- 2.txt --
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
-- 3.txt --
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
Hello, world!
